docs/api/contenttype.md

```diff
@@ -0,0 +1,13 @@
+# Content Type
+
+Adds ability to list supported content types and their primary keys.
+
+## API Endpoints
+
+### GET /api/v1/contenttypes
+
+Returns a list of all allowed content types and their identifiers.
+
+```console
+curl -X GET https://example.com/api/v1/contenttypes
```

Comment:
Can we add example JSON of what a content type object looks like?

Reply:
Added an example in latest commit